### ISSUE TITLE
Add a more realistic API Key example

### ISSUE THREAD

**Issue by caitlinnew** [stoplightio/elements#2360]
## Current Behavior
The current API Key example is simple, as seen below
<img width="665" alt="image" src="https://user-images.githubusercontent.com/92755071/235240540-e8265cd6-0ba5-44a4-8455-2f2d9ef4ddde.png">

## Expected Behavior
A more realistic default example of an API Key and the ability to update the example itself

Example API key: k2KJBhmW5LvzleNv7JKLV1j2brjbUlxpUnV3XCXFBDTc5Dj2vKLkMAd1cAIjSg4g

**Comment by kentbulza** [comment]
Agree. I believe OpenAPI doesn't allow examples here, but would be nice to add a "x-stoplight-example" until OpenAPI supports?
